src/Weather/EDL/StateController.cpp

```
@@ -340,6 +435,25 @@ GetOverlayLabel() noexcept
+
+#ifdef ENABLE_OPENGL
+  if (auto *map = UIGlobals::GetMap()) {
+    const GeoPoint location = map->GetLocation();
+    if (location.IsValid()) {
+      const auto *overlay =
+        dynamic_cast<const EdlMbTilesOverlay *>(map->GetOverlay());
+      if (overlay != nullptr) {
+        double value_mps = 0;
+        if (overlay->SampleAscendancyAt(location, value_mps)) {
+          StaticString<64> with_value;
+          with_value.Format("%s %+.1f m/s", label.c_str(), value_mps);
```

Cppcheck (2.21.0)

[error] Signed integer overflow for expression '0-1'.

(integerOverflow)